### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ViktorAfk**
- [DEMO LINK](https://ViktorAfk.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://ViktorAfk.github.io/layout_product-cards/report/html_report/)
- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️

### CODE REVIEW

**Comment by ViktorAfk** [comment]
I have a problem with a nesting class in stars.scss

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (And678) [inline src/styles/blocks/card.scss:108]:
```suggestion
```

Comment (And678) [reply src/styles/blocks/card.scss:108]: Lots of extra newlines.

File: src/index.html [hunk 2]
```diff
@@ -6,8 +6,50 @@
...
+      </div>
+
+      <div class="card__review">
+        <div class="card__stars card__stars--acive">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
```
Comment (And678) [inline src/index.html:38]: Stars should be a separate BEM block, as described in requirements.
```suggestion
        <div class="stars stars--active">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```

File: src/styles/blocks/stars.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__stars--acive .card__star:not(:last-child) {
+  background-image: url(../images/star-active.svg);
+}
```
Comment (And678) [inline src/styles/blocks/stars.scss:16]: Not done yet:
> all stars--N modifiers work as expected (Highlight first N stars)

**Review by And678 — CHANGES_REQUESTED** [review]
Looks very good but some requirements are not implemented yet. Feel free to join a QnA session if you are having any trouble.

**Review by BudnikOleksii — APPROVED** [review]
Nicely done 🚀

**Comment by ViktorAfk** [comment]
> Nicely done 🚀

**Pull request closed without merging**